src/utilities/templates/flatpages/flatpage-form.html

```diff
@@ -0,0 +1,33 @@
+{% extends "flatpages/base.html" %}
+{% load crispy_forms_tags %}
+
+{% block head %}
+    {{ form.media }}
+{% endblock %}
+
+{% block head_title%}
+
+    {% if not update%}
+        Create Page | 
```

Comment:
This logic should be in the view and passed as a variable, not in the template.  See the Announcements create/update views and form.html as reference.

Reply:
Changed condition to a variable changed by create/update view